include/random_walks/boltzmann_hmc_walk.hpp

```diff
@@ -97,76 +88,40 @@ struct BoltzmannHMCWalk {
             this->settings = settings;
         }
 
-
         /// Samples random points from the spectrahedron from the Boltzmann distribution
         /// \param[in] spectrahedron A spectrahedron
         /// \param[in] interiorPoint A point in the interior of the spectrahedron
         /// \param[in] pointsNum The number of points to sample
         /// \param[out] points The list of the sampled points
         /// \tparam Point class Point with NT and VT as declared above in this class
         template <typename Point>
-        void apply(SPECTRAHEDRON &spectrahedron, Point const & interiorPoint, const unsigned int pointsNum,
+        void apply(ConvexBody &spectrahedron, Point const & interiorPoint, const unsigned int pointsNum,
```

Comment:
the parameter name should be renamed as well, right? From `spectahedron` to `convexbody`